pkg/frontend/openshiftcluster_putorpatch_test.go

```diff
@@ -942,3 +943,375 @@ func TestPutOrPatchOpenShiftCluster(t *testing.T) {
 		})
 	}
 }
+
+// TODO(mjudeiki): This is shameful copy of previous api test. It should be extended
+// when new features lands for new api. and merged with the test above when new
+// api becomes default
+func TestPutOrPatchOpenShiftClusterV20210131preview(t *testing.T) {
```

Comment:
I did a diff of this with `TestPutOrPatchOpenShiftClusterV20200430` and it looks good.